src/components/App.js

```diff
@@ -59,7 +59,7 @@ class App extends Component {
         label)
     }
 
-    let activeThing
+    let activeThing = null
```

Comment:
Question for my own sanity: Is there any time that I do have to include a semicolon and, if so, how will I know?